resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
```

Comment:
So here are the things at play here: 

* I need to get :id as it is now (without the prepended service_) for the directorylist work, so when we are creating the id field in the component dsl, it should not have Service_

* The point of prepending Service_ serves as a measure of documentation. Specifically for me the prepended service makes it clear that this component element contains the service WiX for MCO and not any of the contents of MCO itself. So in this way I like that the component element has Id="Service_MARIONETTECOLLECTIVE" and not just Id="MARIONETTECOLLECTIVE"